ezgui/src/canvas.rs

```diff
@@ -323,9 +325,7 @@ impl Canvas {
             VerticalAlignment::Center => (self.window_height - dims.height) / 2.0,
             VerticalAlignment::Bottom => self.window_height - dims.height,
             // TODO Hack
-            VerticalAlignment::BottomAboveOSD => {
-                self.window_height - dims.height - 60.0 * *assets.scale_factor.borrow()
```

Comment:
changes like this are the motivation for this PR. We no longer have to sprinkle around scale_factor math. Except in exceedingly rare circumstances, all usage of scaling_factor math and physical pixels should live in the graphics backend.

exceptions to this might be things like:
- drawing a "hairline" separator which we want to be as thin as possible
- working with image data, in which case we might need to be aware of physical pixels

Reply:
> drawing a "hairline" separator which we want to be as thin as possible

There are a few separators in `game`, but I think we want them to look proportionally the same on all screens, so this shouldn't be an issue.

> working with image data, in which case we might need to be aware of physical pixels

For better or worse, abst is pure vectors, so not an issue either